source/index.ts

```diff
@@ -52,7 +53,34 @@ const defaults: Partial<Defaults> = {
 		dnsCache: false,
 		useElectronNet: false,
 		responseType: 'text',
-		resolveBodyOnly: false
+		resolveBodyOnly: false,
+		pagination: {
+			parse: response => {
```

Comment:
Maybe this should be named `transform`? As it could be used for other things too. Per: https://github.com/sindresorhus/got/issues/722#issuecomment-493811004

Reply:
Agreed.